### PR TITLE
[feat] - contains?

### PR DESCRIPTION
**Pull request by brandonstubbs**
Please answer the following questions and leave the below in as part of your PR.

- [x] This PR corresponds to an [issue with a clear problem statement](https://github.com/babashka/babashka/blob/master/doc/dev.md#start-with-an-issue-before-writing-code).

- [x] This PR provides tests for code additions and changes

I also fixed an issue on `get` for `Sets`

### CODE REVIEW

**Comment by borkdude** [comment]
I think your changes break this behavior:

Before:

```
$ ./node_cli.js -e '(println (get (js/Map. [[1 nil]]) 1))'
null
```

After:
```
$ ./node_cli.js -e '(println (get (js/Map. [[1 nil]]) 1))'
undefined
```

I'll add a test for that on the main branch.

File: core.js [hunk 1]
```diff
@@ -244,15 +252,15 @@ export function nth(coll, idx) {
 }
 
 export function get(coll, key, otherwise = undefined) {
-  if (coll === null || typeof coll !== "object") return otherwise;
-  if (coll instanceof Set) {
-    return (coll.has(key) && key) || otherwise
-  }
-  if (coll instanceof Map) {
-    return coll.get(key) || otherwise;
+  if (!contains_QMARK_(coll, key)) return otherwise;
```
Comment (borkdude) [inline core.js:255]: I don't think it's great to first check if a collection contains a key and then do the lookup again if it does have that key: this is a 2x performance slowdown

**Comment by borkdude** [comment]
@brandonstubbs Looks good to me. Can you update the core.edn once again?

**Comment by brandonstubbs** [comment]
@borkdude sure done, splitting the symbols in core.edn when you create it would prevent so many conflicts?

**Review by lilactown — APPROVED** [review]
lgtm!

**Comment by borkdude** [comment]
> @borkdude sure done, splitting the symbols to newlines in core.edn when you create it would prevent so many conflicts?

Yes, maybe pretty-printing the symbols works better for git diffs.

**Pull request merged**